packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.Compact.Example.tsx

```diff
@@ -1,7 +1,7 @@
 /* tslint:disable:no-unused-variable */
 import * as React from 'react';
 /* tslint:enable:no-unused-variable */
-import { TextField } from 'office-ui-fabric-react/lib/TextField';
+import { DefaultTextField } from 'office-ui-fabric-react/lib/TextField';
```

Comment:
Why do you change all of the examples to use `DefaultTextField` instead of `TextField`? That doesn't seem right

Reply:
For the same reason all of the Button examples were changed to use `DefaultButton` instead of `Button`.

Reply:
What about compat, when I use TextField today, will I have to change to DefaultTextfield with this change?

Reply:
No, just as with button, the `TextField` component still exists, it will just warn you for using a deprecated component.

Reply:
@lambertwang I don't really think that's a good idea.